README.md

```diff
@@ -373,6 +373,21 @@ api.getCustomersByEmail("test@test.com");
 
 - **email**: String (required)
 
+### api.getAttributes(id, id_type)
+
+Returns a list of attributes for a customer profile.
+
+```javascript
+api.getAttributes("1", IdentifierType.Id);
```

Comment:
I'm not sure we have any typescript in our docs. It would be worth adding a plain JS example for those not using typescript

Reply:
You're correct but I was wondering if we could export the type in the *index.ts* to be usable in Javascript. Enum converted to JS are just plain object, and it would be more consitant.

Reply:
We could, although it requires another export from the `index.ts` file. We could add that here, or just add a stringified version to the docs. I'll leave it up to you

Reply:
Added both to satisfy JS & TS users ! Thanks !